### ISSUE TITLE
display help if snow is run without commands & no docker file is found

### ISSUE THREAD

**Issue by sean-nicholas** [petermikitsh/snow#10]
If you run snow without commands the deploy function is executed. If there is no Dockerfile you end up with an error message. Would be nice to get some hint how to use snow instead. What about using [commander](https://www.npmjs.com/package/commander) or something similar?

**Comment by petermikitsh** [comment]
When I run the latest version of the `secrets` branch in a directory without a Dockerfile, this is the output I see:

```
$ snow
Error finding Dockerfile.
ENOENT: no such file or directory, open '/my/path/foobar/Dockerfile'
```

The command could certainly be clearer. A link to a new usage page, or the existing [snow deploy](https://github.com/snowjs/cli/blob/secrets/docs/commands.md#snow-deploy) documentation from the output would provide for a superior user experience.

**Comment by sean-nicholas** [comment]
Yes, that is the error message I am talking about. Simple link to the docs would be great.
